lib/charms/mongodb/v1/mongodb_provider.py

```diff
@@ -381,11 +395,16 @@ def _get_relations(self, rel=REL_NAME) -> List[Relation]:
         We create users for either direct relations to charm or for relations through the mongos
         charm.
         """
-        return (
-            self.model.relations[MONGOS_RELATIONS]
-            if self.charm.is_role(Config.Role.CONFIG_SERVER)
-            else self.model.relations[rel]
-        )
+        return self.model.relations[self.get_relation_name()]
```

Comment:
You're not using the `rel` argument anymore, so (because it's a private method) you can remove the argument `rel` from the function definition right?

Reply:
I'll do you even one better, lets remove this function and use `self.relation_name` properly